blocks/library/cover-image/index.js

```diff
@@ -146,8 +151,9 @@ registerBlockType( 'core/cover-image', {
 
 		if ( ! url ) {
 			const uploadButtonProps = { isLarge: true };
-			const icon = title ? undefined : 'format-image';
-			const label = title ? (
+			const hasTitle = title && ( ! isArray( title ) || title.length );
```

Comment:
Could [isEmpty](https://lodash.com/docs/#isEmpty) help simplify this?

```js
const hasTitle = ! isEmpty( title );
```

Reply:
Hi @noisysocks, thank you for the feedback. I wanted to catch title as simple strings, isEmpty also handles that case, so I think we can simplify and use it so I applied your suggestion 👍